lib/bulk/common.js

```diff
@@ -35,7 +38,7 @@ var defineReadOnlyProperty = function(self, name, value) {
  * correctly after command execution
  * @ignore
  */
-var Batch = function(batchType, originalZeroIndex) {
+const Batch = function(batchType, originalZeroIndex) {
```

Comment:
since we're already modernizing this code, is it worth converting these to classes? And perhaps giving them better names (e.g. `BatchOperation`)

/cc @daprahamian

Reply:
oh, it looks like the uppercase naming confused me, this isn't really a class is it? I still think the naming could be better here.

Reply:
@mbroadst Would batchOperation or batchState be a better name? Or is just lower-case batch better?

Reply:
Ah right, yes this is a class. I was reverting my request based on things like `Update` and `Delete` below, which seem more like functions